src-plugins/medSegmentation/msegAlgorithmPaintToolbox.cpp

```diff
@@ -978,11 +981,7 @@ void AlgorithmPaintToolbox::updateWandRegion(medAbstractImageView * view, QVecto
     }
 
     if(!view->contains(m_maskAnnotationData))
-    {
         view->addLayer(m_maskAnnotationData);
-        setOutputMetadata(m_imageData, m_maskData);
-        medDataManager::instance()->importData(m_maskData);
-    }
```

Comment:
That was the main problem with this plugin. If you begin to click on the image, you have a random data saved with just this mini-roi on it.